locale/bg-BG/dlgnewedit.dtd

```diff
@@ -15,6 +15,7 @@
 <!ENTITY characters.label "знака">
 <!ENTITY ignoreNumbers.label "Незачитане на промени в цифри">
 <!ENTITY advanced.label "Разширени">
+<!ENTITY requestMethod.label "Метод на заявката">
```

Comment:
Please only include translations that you're 100% sure about. If you're not sure, just don't provide an <!ENTITY> definition and it will be translated by a native speaker at babelzilla.org.